AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m

```diff
@@ -0,0 +1,214 @@
+//
+//  NotificationsPlayerViewController.m
+//  AdvancedPlaybackSampleApp
+//
+//  Created by Michael Len on 11/16/16.
+//  Copyright © 2016 Ooyala, Inc. All rights reserved.
+//
+
+
+#import "NotificationsPlayerViewController.h"
+#import <OoyalaSDK/OoyalaSDK.h>
+#import "AppDelegate.h"
+
+@interface NotificationsPlayerViewController ()
+@property (strong, nonatomic) OOOoyalaPlayerViewController *ooyalaPlayerViewController;
+
+@property NSString *embedCode;
+@property NSString *nib;
+@property NSString *pcode;
+@property NSString *playerDomain;
+@end
+
+@implementation NotificationsPlayerViewController {
+  AppDelegate *appDel;
```

Comment:
yep, that's good. It's either that or `@property (weak, nonatomic) AppDelegate appDel`. I prefer the one you say.